docs/index.yml

```diff
@@ -360,15 +360,15 @@ additionalContent:
       items:
         - title: ".NET Aspire API reference"
           summary: API reference documentation for .NET Aspire
-          url: /dotnet/api?view=dotnet-aspire-8.0&preserve-view=true
+          url: /dotnet/api?view=dotnet-aspire-9.0&preserve-view=true
```

Comment:
@dapine Why not just remove the view altogether so it always points to the latest by default?